internal/driver/onvifdiscovery.go

```diff
@@ -133,10 +133,10 @@ func (d *Driver) createDiscoveredDevice(onvifDevice onvif.Device) (sdkModel.Disc
 		device.Protocols[OnvifProtocol][LastSeen] = time.Now().Format(time.UnixDate)
 		device.Protocols[OnvifProtocol][FriendlyName] = devInfo.Manufacturer + " " + devInfo.Model
 
-		// Spaces are not allowed in the device name
+		//Spaces and slashes are not allowed in the device name
 		deviceName := fmt.Sprintf("%s-%s-%s",
 			strings.ReplaceAll(devInfo.Manufacturer, " ", "-"),
-			strings.ReplaceAll(devInfo.Model, " ", "-"),
+			strings.ReplaceAll(strings.ReplaceAll(devInfo.Model, "/", "-"), " ", "-"), 
```

Comment:
You have an extra blank space at the end of the line here and a few other places. I think `go fmt ./...` should fix them.